### ISSUE TITLE
move the close button to the screen edge side of the tab

### ISSUE THREAD

**Issue by ajagmx** [bwinton/TabCenter#594]
currently to close a tab: 

mouse over the tab center
move mouse to the right to get the close button
click. 

unnecessary and annoying movement since you have to do it on every close tab and you know it's coming each time. if the close button came on the left (or screen-edge) side of the title when the center is expanded, you'd have 

mouse over the tab center
click the close button

better still, why not just drag a tab off the center to close it - without expanding the bar. that would be good for touch users also.

**Comment by MrJustMe** [comment]
Even better i would prefer the close button inside the active tab on the top right side... Both methods coud be great i think.

**Comment by MrJustMe** [comment]
You don't have to make the click target small, better than that, make the zone for the blue bar, (the one that point the active tab), bigger and place the close button there. The currently way is harder for close the tab for sure, the first time i test this feature, i quitted in 2 hours for this reason. I also will add another button in the current open and active tab, i mean inside the open tab/window to close it. It will be more confortable even... and sorry for my english, it is not good enough i think.

**Comment by E100Beta** [comment]
Agree, right now closing a tab with a mouse is **infuriating**. I even started using Ctrl+W to close tabs. Even worse, I can't think of any good way to fix this. Every fix has its downside.

- The x can be moved to the side, but, as pointed before, it will increase error rate.
- Drag-from-center-to-kill, suggested in the first post, conflicts with behavior of moving tab to separate window.
- The way that I think solves the problem, but makes the interface more vertical, is to move the close button to be above the miniature. Obviously, it needs to be smaller. I imagine it would be not as pretty, but it would be more effective. It can also be an _optional_ thing, because, for example, my friend likes keeping every website on its own tab and browse it only on that tab. I like opening every individual page of the site on the separate tab. So, my friend probably wouldn't even notice the problem of efficiency of tab closing. I, on the other hand...
- Other thing that can help this issue (and I don't know, maybe this already exists?) is to make a shortcut to open the Center. For example, I am about to finish reading and close the tab. I press Tab (which is the best button for this binding, because tabs, Tab Center, and it's on the left), the Center opens, i click the close button, stop holding Tab, and the Center hides back.

**Comment by bwinton** [comment]
`Ctrl-shift-K` (or `Ctrl-shift-L` in an upcoming version) will toggle the sidebar open and closed…
(`Cmd-shift-K/L` on Mac.)

**Comment by MrJustMe** [comment]
I like the button above the miniature and more over the blue bar (the blue bar who indicates the active tab could have a little more width), anyway a close button in the tab bar is not so difficult i think, the thing is it will requiere some re-design in the tab bar...

**Comment by E100Beta** [comment]
@bwinton Thank you! I already accidentally found it while looking through PR history. You should really put it somewhere where more people will see it. While not really fixing the issue, this makes using the addon much more comfortable for me.

**Comment by bwinton** [comment]
Yeah, the issue is trickier to fix than it first seems, and we definitely need to publicize the hotkeys a little more…

**Comment by indrekln** [comment]
Suggestions:
Option 1: add a hotkey (Ctrl) to display the X for closing.
Option 2: justify the X to the right of the tab field, so that dragging the sidebar narrower would bring it closer to the screen capture image.

The toggle for open/closed position is great, but then it would require the X to be right next to the screen capture in the closed position.

I prefer to drag the size of the tab bar to display only the screen capture and the X becomes hidden. Middle-clicking to close tabs is fine, unless you are using a laptop with only a touchpad and no middle-key - then its annoying.
